Tests/AMPStorageTests.m

```diff
@@ -23,17 +31,15 @@ - (void)setUp {
 
 - (void)tearDown {
     [super tearDown];
-    NSString* dir = [AMPStorage getAppStorageAmpDir:@""];
```

Comment:
The previous code wouldn't work because adding `if (instanceName == NULL || [instanceName length] == 0) { ` logic in the getAppStorageAmpDir function means it will always return the directory for $default_instance instead of the root directory. So it won't clean up everything.  @dantetam